### ISSUE TITLE
WebImage is not rendered with drawingGroup()

### ISSUE THREAD

**Issue by michala-pro** [SDWebImage/SDWebImageSwiftUI#314]
Example code to reproduce:

```swift
struct AsyncImageDemoView: View {
    let url = URL(string: "https://placekitten.com/200/300")
    
    var body: some View {
        if let url {
            Text("URL: \(url)")
            WebImage(url: url)
                .drawingGroup()
        }
    }
}
```

**Comment by dreampiggy** [comment]
I guess this issue was solved  in 3.0.4 via #316 

![image](https://github.com/SDWebImage/SDWebImageSwiftUI/assets/6919743/4ae7cf07-6546-4590-bf8c-fc9937936fe3)

**Comment by michala-pro** [comment]
It works in 3.0.4 - thank you!
